app/controllers/course/assessment/assessments_controller.rb

```diff
@@ -82,7 +82,9 @@ def load_category
                   current_course.assessment_categories.new.tap do |c|
                     c.assign_attributes(title: t('course.assessment.assessments.index.header'),
                                         weight: 0)
-                    c.tabs.build(title: t('course.assessment.assessments.index.header'), weight: 0)
+                    c.tabs.build(title: t('course.assessment.assessments.index.header'),
+                                 weight: 0,
+                                 category: c)
```

Comment:
The next thing that I need your help to design is how to find a way to initialise this by default when the user creates the course.